test/fixtures/command_line/bwb.xcodeproj/xcshareddata/xcschemes/LibSwiftTests.__internal__.__test_bundle.xcscheme

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
```

Comment:
We should set `version = "1.3"` here.

Reply:
On `<Scheme>` not `<?xml?>`, in case it wasn't clear.

Reply:
Done.